jsonapi_test.go

```diff
@@ -327,3 +329,14 @@ type ArticleDoubleID struct {
 	Title   string `jsonapi:"attribute" json:"title"`
 	OtherID string `jsonapi:"primary,article"`
 }
+
+type Metadata struct {
+	LastModified time.Time `jsonapi:"attribute" json:"lastModified"`
+}
+
+type ArticleEmbedded struct {
+	Metadata
```

Comment:
If changed to an embedded pointer type, the attribute is excluded. Wonder if we can fix that?